pallets/proposals/src/impls.rs

```diff
@@ -23,7 +23,7 @@ impl<T: Config> Pallet<T> {
     }
 
     pub fn project_account_id(key: ProjectKey) -> ProjectAccountId<T> {
-        T::PalletId::get().into_sub_account_truncating(key)
+        T::PalletId::get().into_sub_account_truncating(format!("//{key}"))
```

Comment:
This looks like a breaking change to me. 
If a project exists already, then its account id cannot be generated anymore due to the // prefix.

Reply:
The intention is to avoid the case that `project_account_id()` and `account_id()` of the pallet returns the same value. (when key = 0)

Reply:
```rust
pub fn account_id() -> T::AccountId {
        T::PalletId::get().into_account_truncating()
}
```